www/include/reporting/dashboard/viewServicesLog.php

```diff
@@ -45,10 +45,8 @@
 /*
  *  Getting service to report
  */
-isset($_GET["host_id"]) ? $host_id = $_GET["host_id"] : $host_id = "NULL";
-isset($_POST["host_id"]) ? $host_id = $_POST["host_id"] : $host_id;
-isset($_GET["item"]) ? $service_id = $_GET["item"] : $service_id = "NULL";
-isset($_POST["item"]) ? $service_id = $_POST["item"] : $service_id;
+$host_id = filter_var($_GET['host_id'], FILTER_VALIDATE_INT) ?? filter_var($_POST['host_id'], FILTER_VALIDATE_INT);
```

Comment:
if $_REQUEST['host_id'] is not sent, it will log a notice (future fatal) as it won't be defined
please add a default value like here https://github.com/centreon/centreon/pull/8419/files#diff-23497cf3c24512e5972599132fbe2244R51

Reply:
fix with https://github.com/centreon/centreon/pull/8419/commits/23185b9036383d869664a0590531e69abe7f06c2